common/core/desktop/tests/unit/kmx/028 - smp.kmn

```diff
@@ -4,7 +4,6 @@ c expected: \U0001F601yx
 c context:
 
 store(&version) '10.0'
-store(&name) '028smp'
```

Comment:
Why this change?  It seems "thrown in."

Reply:
See original details above:
> * Corrects a bug that was identified by the 15.0 version of kmcomp in one unit test (duplicate &NAME stores -- the meson test script prepends a &NAME store to each source file during build).

Reply:
We've all apparently been building these keyboards with the 14.0 compiler until now.